sandbox/grommet-app/src/themes/theme.jsx

```diff
@@ -2649,9 +2682,6 @@ const buildTheme = (tokens, flags) => {
         },
       },
     },
-    buttonGroup: {
```

Comment:
This is not part of Grommet theme object, is just something we have locally in aries theme. Removing so it doesn't end up in grommet-theme-hpe.